compiler/rustc_data_structures/src/tagged_ptr/copy/tests.rs

```diff
@@ -25,6 +26,24 @@ fn smoke() {
     assert!(ptr::eq(copy.pointer(), reference));
 }
 
+#[test]
+fn stable_hash_hashes_as_tuple() {
+    let hash_packed = {
+        let mut hasher = StableHasher::new();
+        tag_ptr(&12, Tag2::B11).hash_stable(&mut (), &mut hasher);
```

Comment:
Given the previous incident here, I'd prefer seeing fully qualified syntax instead of method calls

Reply:
I don't think it matters, if the incident ever happens again the test will just fail.